Database-Files/CreateDatabase.sql

```diff
@@ -39,15 +39,14 @@ CREATE TABLE capLead (
 DROP TABLE IF EXISTS authData;
 CREATE TABLE authData (
     ID INT PRIMARY KEY AUTO_INCREMENT,
-    username VARCHAR(50) NOT NULL,
+    username VARCHAR(50) NOT NULL UNIQUE,
     passwordHash VARCHAR(60) NOT NULL,
-    salt VARCHAR(50) NOT NULL,
-    failedAttempts INT NOT NULL,
-    lockedOut BOOL NOT NULL,
-    lockoutDate DATETIME NOT NULL,
-    jwt VARCHAR(50) NOT NULL,
-    jwtDate DATETIME NOT NULL,
-    isAdmin BOOLEAN NOT NULL
+    failedAttempts INT NOT NULL DEFAULT 0,
```

Comment:
Why have we removed the salt field?

Reply:
dont need to store the salt because of the bcrypt implementation